packages/cli/src/databases/entities/MessageEventBusDestinationEntity.ts

```diff
@@ -0,0 +1,12 @@
+import { MessageEventBusDestinationOptions } from 'n8n-workflow';
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { AbstractEntity, jsonColumnType } from './AbstractEntity';
+
+@Entity({ name: 'eventdestinations_entity' })
```

Comment:
if we rename the table to `event_destinations`, we can change this to `@Entity()`

Reply:
_entity is added by the ORM iirc

Reply:
since many of our tables don't have that suffix, I believe this yet another case of copy-pasting. TypeORM doesn't have any implicit suffix.